client/src/components/RelatedProducts/Cards/Individualcard.jsx

```diff
@@ -20,15 +24,19 @@ function Individualcard({ product }) {
             $
             {product[0].default_price}
           </p>
-          <p>Stars</p>
+          <div>
+            <Star score={starRating} />
+            {' '}
```

Comment:
Is this space intended?

Reply:
Linter was throwing an error and I just did autofix and it created it.